presto-orc/src/main/java/com/facebook/presto/orc/cache/StorageOrcFileTailSource.java

```diff
@@ -42,21 +43,21 @@
 {
     private static final Logger log = Logger.get(StorageOrcFileTailSource.class);
 
-    private static final int EXPECTED_FOOTER_SIZE = 16 * 1024;
-
     private static final int CURRENT_MAJOR_VERSION = 0;
     private static final int CURRENT_MINOR_VERSION = 12;
+    private static final int MINIMUM_TAIL_SIZE = 256; // max postscript size(255) + 1 byte post script length
 
     private final boolean dwrfStripeCacheEnabled;
     private final int expectedFooterSize;
 
     public StorageOrcFileTailSource()
     {
-        this(EXPECTED_FOOTER_SIZE, false);
+        this(EXPECTED_FOOTER_SIZE_BYTES, false);
     }
 
     public StorageOrcFileTailSource(int expectedFooterSize, boolean dwrfStripeCacheEnabled)
```

Comment:
We usually say `expectedFooterSizeInBytes`. Please also fix all other places using this variable name